helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -3,13 +3,13 @@
 {{- else if eq .Capabilities.KubeVersion.Minor "22" -}}
   {{- $_ := set . "cloudProviderAzureVersion" "v1.1.27" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "23" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.23.27" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.23.28" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "24" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.24.14" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.24.15" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "25" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.25.8" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.25.9" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "26" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.26.4" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.26.5" }}
```

Comment:
have you verified that all images are published and available?

Reply:
Yes, all the pipeline builds passed. I hadn't done a `docker pull`, but I just did and the images are there.